contracts/stellar-interchain-token-service/src/contract.rs

```diff
@@ -224,6 +224,8 @@ impl InterchainTokenServiceInterface for InterchainTokenService {
         env: &Env,
         token_address: Address,
     ) -> Result<BytesN<32>, ContractError> {
+        token_metadata::token_metadata(env, &token_address, &Self::native_token_address(env))?;
```

Comment:
add comment to explain that this is used for validating the token address and metadata. Use `let _ = ...` to be explicit that return value is intentionally ignored